src/openms_gui/source/VISUAL/APPLICATIONS/TOPPViewBase.cpp

```diff
@@ -134,7 +134,7 @@ namespace OpenMS
   /// supported types which can be opened with File-->Open
   const FileTypes::FileTypeList supported_types({ FileTypes::MZML, FileTypes::MZXML, FileTypes::MZDATA, FileTypes::SQMASS,
                                                   FileTypes::FEATUREXML, FileTypes::CONSENSUSXML, FileTypes::IDXML,
-                                                  FileTypes::DTA, FileTypes::DTA2D,
+                                                  FileTypes::DTA, FileTypes::DTA2D, FileTypes::MGF, FileTypes::MS2,
```

Comment:
Cool! Can we add MSP for the NIST databases?
https://github.com/OpenMS/OpenMS/blob/develop/src/openms/source/FORMAT/FileHandler.cpp#L369

Or was something speaking against?

Reply:
It can not yet be read via FileHandler. I will take a look.